### ISSUE TITLE
Support for Field Alias (part of GraphQL Specification)

### ISSUE THREAD

**Issue by PabloSzx** [supabase/pg_graphql#40]
**Describe the bug**
As discovered while trying to use [gqty](https://github.com/gqty-dev/gqty) with pg_graphql, it doesn't seem to respect field alias, which is of course very important part of the graphql specification https://spec.graphql.org/June2018/#sec-Field-Alias

https://github.com/gqty-dev/gqty/issues/520#issuecomment-1012612204

**To Reproduce**

You can re-use the reproduction repo made originally for the issue made in gqty: https://github.com/gqty-dev/gqty/issues/520

**Expected behavior**

To support field aliases

**Screenshots**

![image](https://user-images.githubusercontent.com/8672915/149424712-f6f0c0a2-bdd9-47ba-92d4-6d2772b0e550.png)
![image](https://user-images.githubusercontent.com/8672915/149424746-35eb5db9-7400-42e0-b68b-611d020faa65.png)

the graphql request clearly requires that allMfes is served in allMfes0, which is not what it's getting

**Comment by olirice** [comment]
thanks!

This bug was limited to top level keys. It is resolved on master at or after 119b613135a855762623d10f58f6a548d021d6e7
please let me know + reopen if that does not resolve the issue

**Comment by cdaringe** [comment]
Hey @olirice, as always, thanks for the support! Unfortunately, the bug persists, even after pulling latest with your patches from yesterday. Nonetheless, the reproduction is super easy. 

- on master@latest, run docker compose
- open up graphiql
- send `query{allAccounts0:allAccounts{__typename edges{__typename node{__typename id}}}}`

observe the alias not respected

**Comment by olirice** [comment]
I think you may be running into a docker cache issue. Here's a screenshot

<img width="876" alt="Screen Shot 2022-01-15 at 3 33 03 PM" src="https://user-images.githubusercontent.com/12958657/149638389-31d8e41f-85ea-4792-af2d-15f3559396c2.png">

try running
```sh
docker-compose down -v
docker rmi pg_graphql_db
```
and then
```sh
docker-compose up
```

**Comment by cdaringe** [comment]
Yep. My bad. I always prune the containers, forget to rebuild the _image_!. Thx
